src/Servers/testassets/TestSites/ServerComparison.TestSites.csproj

```
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
```

```suggestion
  <PropertyGroup>
    <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;net461</TargetFrameworks>
```

We don't need to change TFMs on test projects conditionally anymore. I think this was a workaround from a long time ago.